src/org-manager/containers/pending-organisations/pending-organisations.component.ts

```diff
@@ -21,7 +22,7 @@ export class PendingOrganisationsComponent extends OrganisationListComponent {
     return fromStore.getPendingOrganisationsArray;
   }
 
-  constructor(protected readonly store: Store<fromStore.OrganisationRootState>) {
-    super(store);
+  constructor(protected readonly store: Store<fromStore.OrganisationRootState>, route: ActivatedRoute) {
```

Comment:
Same comment as earlier - is route being used, does it need to be in the super() and should be private readonly.

Reply:
it needs to pass to superclass cant add private but we can add protected instead of private because angular default accessor is protected

Reply:
You've changed to protected readonly now, so resolved.